src/Okta.Sdk/Generated/IUserActivationToken.Generated.cs

```diff
@@ -13,6 +13,7 @@
 
 namespace Okta.Sdk
 {
+    /// <summary>Contains the method contract for UserActivationToken resources.</summary>
```

Comment:
Since we name our class `IUserActivatationToken`, should the documentation be prepended with an `I`?

Or.. lets make this more clear saying it is an interface for the `UserActivationToken` class.